packages/cli/src/bin/ceramic.ts

```diff
@@ -24,7 +24,10 @@ program
     '--state-store-s3-bucket <string>',
     `The S3 bucket name to use for state store. If not provided data will only be saved locally but not to S3. Deprecated.`
   )
-  .option('--gateway', 'Makes read only endpoints available. It is disabled by default')
+  .option(
```

Comment:
Given that this is a breaking release, do you think we need to keep `--gateway` around at all?

Reply:
I'm in favour of removing it